updating/updating-cluster-rhel-compute.adoc

```diff
@@ -42,4 +42,6 @@ include::modules/rhel-compute-using-hooks.adoc[leveloffset=+2]
 
 include::modules/rhel-compute-available-hooks.adoc[leveloffset=+2]
 
+include::modules/update-upgrading-mcp.adoc[leveloffset=+1]
```

Comment:
RHEL worker is using ansible playbook to run upgrade, this should not be linked here.

Reply:
Removed.